stylesheets/_component.rules.scss

```diff
@@ -0,0 +1,271 @@
+        top: 10px;
+        right: 10px;
+    }
+
+    [href].conditional-block__link--when {
+        color: #1292C9;
+    }
+
+    [href].conditional-block__link--and {
+        color: #646464;
+    }
+
+    [href].conditional-block__link--then {
+        color: #459A8C;
+    }
+}
+
+.rule-block-wrapper ~ .rule-block-wrapper {
+    margin-top: -1px;
+}
+
+.rule-block {
+    background: color(white);
+    border: 1px solid #D1D1D1;
+    box-shadow: 0px 2px 4px 0px rgba(0,0,0,0.10);
+    //overflow: hidden;
+    padding: 0;
+    position: relative;
+    width: 100%;
+    z-index: 1000;
+
+    @include respond-min($screen-tablet) {
+        display: table;
+        padding: 0 15px 0 0;
+    }
+}
+
+.rule-block--and {
+    margin-top: 24px;
+}
+
+.rule-block--indented {
+    border-radius: 0;
+    margin-left: 20px;
+}
+
+.rule-block--new {
+    animation-duration: 1s;
+    animation-name: blueFadeToWhite;
+
+    .rule-block {
+        animation-duration: 1s;
+        animation-name: blueFadeToWhite;
+    }
+
+    .rule-block__header {
+        animation-duration: 1s;
+        animation-name: blueFadeToGrey;
+    }
+}
+
+.rule-block__header {
+    background: #F5F5F5;
+    border-bottom: 1px solid #D1D1D1;
+    display: block;
+
+    @include respond-min($screen-tablet) {
+        border-right: 1px solid #D1D1D1;
+        border: 0;
+        display: table-cell;
+        height: 100%;
+        padding: 0 10px;
+        text-align: center;
+        vertical-align: middle;
+        width: 95px;
+    }
+}
+
+.rule-block__header--when {
+    box-shadow: inset 4px 0px 0px 0px #128FC5;
+    color: #128FC5;
+}
+
+.rule-block__header--then {
+    box-shadow: inset 4px 0px 0px 0px #459A8C;
+    color: #459A8C;
+}
+
+.rule-block__header--and,
+.rule-block__header--or{
+    box-shadow: inset 4px 0px 0px 0px #646464;
+    color: #646464;
+}
+
+.rule-block__heading {
+    display: inline-block;
+    font-size: 1.2em;
+    margin: 0;
+    padding: .5em 0 .5em .7em;
+    text-transform: capitalize;
+
+    @include respond-min($screen-tablet) {
+        padding: 1.3em 0;
+        text-align: center;
+    }
+}
+
+.rule-block__heading__icon {
+    font-size: 16px;
+    margin-left: 5px;
+}
+
+.rule-block__controls {
+    margin: 1em 0;
+
+    @include respond-min($screen-tablet) {
+        margin: 1em 1.7em 0 .4em;
+    }
+}
+
+.rule-block__control {
+    display: block;
+    margin: 0 10px 10px;
+
+    @include respond-min($screen-tablet) {
+        display: inline-block;
+        margin: 0 0 .5em .4em;
+    }
+
+    .control__label {
+        padding: 0;
+    }
+}
+
+.conditional-block {
+    position: relative;
+}
+
+.conditional-block--flush {
+    // margin: 24px 0 24px 58px;
+    margin: 24px 0 0 58px; // seeing whether layout still works without bottom margin
+
+    @include respond-min($screen-tablet) {
+        // margin: 24px 0 24px 68px;
+        margin: 24px 0 0 68px; // seeing whether layout still works without bottom margin
+    }
```

Comment:
These seem TODOish comments, should they be in a release? I've got a feeling they are probably no longer relevant.